service/src/test/java/com/ge/predix/acs/monitoring/MockCacheHealthCheckUtil.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 General Electric Company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.ge.predix.acs.monitoring;
+
+public final class MockCacheHealthCheckUtil {
+
+    public static final String RESOURCE = "resource";
+    public static final String SUBJECT = "subject";
+    public static final String DECISION = "decision";
+
```

Comment:
We could move these variables into respective cache health indicators.  No need for this class.